packages/mdc-typography/README.md

```diff
@@ -33,7 +34,7 @@ MDC Typography is a foundational module that applies these styles to MDC Web com
   <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
 </head>
 <body class="mdc-typography">
-  <h1 class="mdc-typography--display4">Big header</h1>
+  <h1 class="mdc-typography--headline1">Big header</h1>
```

Comment:
Suggestion: can we pull out this entire top "usage" section? It's completely redundant of the one below, and the one below is more in line with our README template.